server.sh

```diff
@@ -1 +1,23 @@
-python -m uvicorn main:app --reload --host 0.0.0.0 --port 8000
\ No newline at end of file
+#!/bin/bash
+
+config_path=""
+
+helpFunction()
+{
+    echo "Usage $0 -c \"config_path\""
+    exit 1
+}
+
+while getopts "c:" opt
+do
+    case "$opt" in
+        c) config_path=$OPTARG ;;
+        *) echo "Invalid option" ; helpFunction ;;
+    esac
+done
+
+if [ -z "$config_path" ]; then
+    python -m uvicorn main:app --host 0.0.0.0 --port 8000
```

Comment:
How about defining variables for host address and port number in the shell script? For convenient modification of the script.